all-the-icons-ivy-rich.el

```diff
@@ -991,7 +991,7 @@ Return `default-directory' if no project was found."
       (projectile-project-root))
      ((fboundp 'project-current)
```

Comment:
I didn't add a check here for whether `(fboundp 'project-root)` since the existence of one implies the other, and they're both built-in anyway. But if you'd rather have the safety of checking, we can easily look for both.

Reply:
Yes. It's better add the checking like this, for backward compatibility. 

```elisp
(if (fboundp 'project-root)
     (project-root project)
   (cdr project))
```

Reply:
You're totally right, I had thought `project-root` had been around from the start (hence we don't really need to guard it), but it's actually only a few years old. Once upon a time, it was `project-roots` which returned a list, that tripped me up. This is a good check 👍